brave/instrumentation-aws-java-sdk-core/src/test/java/brave/instrumentation/aws/AwsClientTracingTest.java

```diff
@@ -98,6 +99,15 @@ public void testInternalAwsRequestsDoNotThrowNPE() throws InterruptedException {
 
     s3Client.doesBucketExistV2("Test-Bucket");
 
+    // The HEAD request is also recorded
+    MutableSpan errorSpan = testSpanHandler.takeRemoteSpanWithError(brave.Span.Kind.CLIENT);
```

Comment:
@codefromthecrypt this is the only change that looks is bit off: previously, the HEAD request was not the part of the trace (I haven't looked why but I surely could), now it is - and to be fair, it looks like it should be, what do you think?

Reply:
Yeah I agree. I think that it wasn't picked up prior was an anomaly possibly explainable through dep changes, but in any case, your addition makes sense.